src/ray/protobuf/common.proto

```diff
@@ -760,3 +760,15 @@ message NamedActorInfo {
   string ray_namespace = 1;
   string name = 2;
 }
+
+// Info about an object's location.
+// Currently used for metric exporting only.
+enum ObjectLocation {
+  // TODO(rickyx): seperate fallback memory from this enum
+  // Shared memory sealed + fallback memory sealed
+  IN_MEMORY = 0;
+  // Spilled to external storage
+  SPILLED = 1;
+  // Unsealed memory
+  UNSEALED = 2;
+}
```

Comment:
new line!

Reply:
there is actually no new line if you hover on the icon.

Reply:
oh lol there needs to be a new line...